adding database

wulalasuperdope opened the pull request:
- fetch database branch
- create "climbAR" database in Postico
- run "npm i"
- run the app

nries1 commented:
Awesome @wulalasuperdope 
I made one change:
I added a RouteModel table, which maps routes to holds. Now, the hold table only contains the hold type and the hold id. When an admin creates a climbing route. All of the holds that they added to it will be SELECTED FROM the hold table. And all of their selections will go into the routeModel table, which will have the x,y,z coordinates for that hold AT THAT route.

I did this because x,y,z coordinates depend on a hold within a given route. Not on the hold itself. So holds themselves don't have coordinates until they get dragged onto a route.

wulalasuperdope commented:
> Awesome @wulalasuperdope
> I made one change:
> I added a RouteModel table, which maps routes to holds. Now, the hold table only contains the hold type and the hold id. When an admin creates a climbing route. All of the holds that they added to it will be SELECTED FROM the hold table. And all of their selections will go into the routeModel table, which will have the x,y,z coordinates for that hold AT THAT route.
> 
> I did this because x,y,z coordinates depend on a hold within a given route. Not on the hold itself. So holds themselves don't have coordinates until they get dragged onto a route.

Yes I agree with you. I added a few more hold types just in case.
